x-pack/plugins/monitoring/public/components/enterprise_search/overview/overview.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { PureComponent } from 'react';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiScreenReaderOnly,
+  EuiPanel,
+  EuiSpacer,
+  EuiFlexGrid,
+  EuiFlexItem,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+// @ts-ignore
+import { MonitoringTimeseriesContainer } from '../../chart';
+import { Status } from './status';
+
+export class EnterpriseSearchOverview extends PureComponent {
```

Comment:
Can we just use a functional component here? `React.memo` maybe?

Reply:
Addressed here: 3d48164da9990bd4cccbfcd2ae7d3779bc7778cd